docs/upload/storage-adapters.mdx

```diff
@@ -100,6 +100,12 @@ export default buildConfig({
         'media-with-prefix': {
           prefix,
         },
+        'media-with-presigned-downloads': {
+          // Filter only mp4 files
+          shouldUseSignedURL: ({ collection, filename, req }) => {
```

Comment:
This property should be nested under `signedDownloads`, shouldn't it?

This is what you have in the test config:
```
          signedDownloads: {
            shouldUseSignedURL: (args) => {
              return args.req.headers.get('X-Disable-Signed-URL') !== 'true'
            },
          },
          ```